### PR TITLE
lab 1 code

### PR DESCRIPTION
**Pull request by abbymartin**

### CODE REVIEW

File: AbbyM/Lab1/src/hello_world.cpp [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#include "../include/hello_world.h"
```
Comment (RCMast3r) [inline AbbyM/Lab1/src/hello_world.cpp:1]: please do not use relative includes, see here about how cmake handles your include path setup for you:

https://cliutils.gitlab.io/modern-cmake/chapters/basics.html#targets-are-your-friend

File: AbbyM/Lab1/src/main.cpp [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#include "hello_world.cpp"
```
Comment (RCMast3r) [inline AbbyM/Lab1/src/main.cpp:1]: please do not include source files, only include header files.

File: AbbyM/Lab1/include/hello_world.h [hunk 3]
```diff
@@ -0,0 +1,11 @@
+#ifndef hello_world_h
+#define hello_world_h
+
+#include <string>
+
```
Comment (RCMast3r) [inline AbbyM/Lab1/include/hello_world.h:5]: where is your iostream include? also why include string when you dont use it?

please include what you use: https://www.fluentcpp.com/2021/01/01/include-what-you-use/

File: AbbyM/Lab1/CMakeLists.txt [hunk 4]
```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.15...3.30)
+
+project(HelloWorldLab1 VERSION 1.0
+                  DESCRIPTION "lab 1 hello world"
+                  LANGUAGES CXX)
+
+add_library(HelloWorldLibrary STATIC src/hello_world.cpp include/hello_world.h)
+add_executable(HelloWorld src/main.cpp)
+
```
Comment (RCMast3r) [inline AbbyM/Lab1/CMakeLists.txt:9]: please make sure you point your target to your include's folder as linked to in my other comment

**Review by RCMast3r — CHANGES_REQUESTED** [review]

File: AbbyM/Lab1/CMakeLists.txt (hunk 4 above)
Comment (abbymartin) [reply AbbyM/Lab1/CMakeLists.txt:11]: all changes should be fixed

**Review by RCMast3r — APPROVED** [review]
next time pls separate your labs / parts into separate PRs

**Pull request merged**